README.md

```diff
@@ -120,18 +120,19 @@ you must await them.
 By default, the timeout for assertions is set to 5 seconds, and the polling
 interval is set to 100 milliseconds.
 
-| Assertion                            | Description                                           |
-| ------------------------------------ | ----------------------------------------------------- |
-| `toBeChecked(opts?)`                 | Element is checked                                    |
-| `toBeDisabled(opts?)`                | Element is disabled                                   |
-| `toBeEditable(opts?)`                | Element is editable                                   |
-| `toBeEnabled(opts?)`                 | Element is enabled                                    |
-| `toBeHidden(opts?)`                  | Element is hidden                                     |
-| `toBeVisible(opts?)`                 | Element is visible                                    |
-| `toContainText(text, opts?)`         | Element contains text.                                |
-| `toHaveAttribute(attribute, value?)` | Element has specific attribute and, optionally, value |
-| `toHaveText(text, opts?)`            | Element has text.                                     |
-| `toHaveValue(value)`                 | Element has specific value                            |
+| Assertion                            | Description                                                             |
+| ------------------------------------ | ----------------------------------------------------------------------- |
+| `toBeChecked(opts?)`                 | Element is checked                                                      |
+| `toBeDisabled(opts?)`                | Element is disabled                                                     |
+| `toBeEditable(opts?)`                | Element is editable                                                     |
+| `toBeEmpty(opts?)`                   | Input element value is empty, otherwise non-input text content is empty |
```

Comment:
Nit; I'm not sure if I got this correctly, fully, until I read the function description in `expectRetrying.ts`. So, perhaps I'd suggest to leave a short description, following the convention of the other methods, like `Element is hidden`, and leave it to the reader to go deeper if needed 🤔

Reply:
Sure, and to be consistent 👍 Done in 478d984514f5a610cb50af5da12b00b1e8d04ab0